wiki/Modding/Nomination/en.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
```

Comment:
could add bubble to tags? noms are still referred as bubbles sometimes

Reply:
alright